styles/_typography.scss

```diff
@@ -57,3 +56,19 @@ code {
   color: var(--brand-tertiary);
   font-weight: bold;
 }
+
+
+@media only screen and (min-width: 600px) {
```

Comment:
For consistency we should probably use the media query mixins for this?

Reply:
Thanks @andrewebdev. Forgot to update that one with the mixins.